lib/apimatic-core/logger/nil_sdk_logger.rb

```diff
@@ -1,7 +1,7 @@
 module CoreLibrary
   # This class is nil implementation of ApiLogger.
   class NilSdkLogger < ApiLogger
-    def initialize(logging_config: LoggingConfiguration.new); end
+    def initialize(logging_config: ApiLoggingConfiguration.new); end
```

Comment:
There is no need to take any parameter in the constructor of the `NilSdkLogger`.